### ISSUE TITLE
[Support]: Supervisor warnings after Frigate uninstall, any way to remove fully?

### ISSUE THREAD

**Issue by crnah2o** [blakeblackshear/frigate#2800]
### Describe the problem you are having

Hi. 

First of all, many thanks for creating such a great piece of software. 

I've tested it on my personal home assistant installation for about a month now, both the official and beta version(s) and the time has come for me to set it up on the "production" home assistant os installation for my girlfriend. I've uninstalled everything that was available to me through the home assistant "gui" and yet the supervisor still throws warnings  in the log about frigate demanding full device access. Is there any way for me to remove every trace of frigate from my installation of home assistant?

Running latest home assistant os on an intel nuc.

Thanks in advance!

### Version

no frigate installed

### Frigate config file

```yaml
no frigate installed
```

### Relevant log output

```shell
22-02-13 11:34:50 WARNING (SyncWorker_2) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR (Full Access) Beta (0.10.0)
22-02-13 11:34:50 WARNING (SyncWorker_2) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR (Full Access)
22-02-13 11:34:50 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/core repository
22-02-13 11:34:50 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/ccab4aaf repository
22-02-13 11:34:50 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/a0d7b954 repository
22-02-13 11:34:51 WARNING (SyncWorker_2) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR (Full Access) Beta (0.10.0)
22-02-13 11:34:51 WARNING (SyncWorker_2) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR (Full Access)
22-02-13 11:34:51 INFO (MainThread) [supervisor.store] Loading add-ons from store: 67 all - 67 new - 0 remove
22-02-13 11:34:51 INFO (MainThread) [supervisor.addons] Found 8 installed add-ons
22-02-13 11:34:51 INFO (SyncWorker_3) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-zwave_js with version 0.1.53
22-02-13 11:34:51 INFO (SyncWorker_6) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/tor/amd64 with version 3.0.5
22-02-13 11:34:51 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/ssh/amd64 with version 10.0.2
22-02-13 11:34:51 INFO (SyncWorker_5) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-samba with version 9.5.1
22-02-13 11:34:51 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/node-red/amd64 with version 11.0.0
22-02-13 11:34:51 INFO (SyncWorker_4) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-configurator with version 5.3.3
22-02-13 11:34:51 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-check_config with version 3.9.0
22-02-13 11:34:51 INFO (SyncWorker_7) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/adguard/amd64 with version 4.4.4
```

### FFprobe output from your camera

```shell
no cameras installed
```

### Frigate stats

```json
no frigate installed
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

no cameras installed

**Comment by blakeblackshear** [comment]
This sounds like a HassOS issue. I'm not sure why you would be seeing that log message if you have removed everything.

**Comment by nickwild-999** [comment]
I'm having this issue also

**Comment by crnah2o** [comment]
> I'm having this issue also

Ah, so I'm not alone! I'll post my/our issue on the HA forums and link it here for you, just so you can join in if it suits you.

**Comment by crnah2o** [comment]
> This sounds like a HassOS issue. I'm not sure why you would be seeing that log message if you have removed everything.

FYI: The warnings disappear when Frigate repository is removed from "addon store" and comes back when repository is added back (without installing any of the frigate addons).

**Comment by Sniperson2002** [comment]
I recently found myself in the same boat, after banging my head against the wall looking through hacs, the config, automations, ect I started searching and found this thread as well as the other mentioned above. I have the erroneous mqtt messages from frigate still flowing through as well. I have removed the repo and it is no longer installed. I've done a full reboot after double checking everything thinking it may have a script running rogue.  I have also checked in portainer for a container for frigate with no luck. I am at the point of reinstalling and trying to enable/disable the mqtt notifier. Any other ideas minus a reinstall? Thanks in advance.
